### PR TITLE
Add views

### PR DESCRIPTION
**Pull request by jonas-45**
1. In this milestone, I used preprocessed HTML with embedded Ruby code to create views for both post and user controllers
2. The index and show actions for both user and post controllers are implemented to display content in the views.
3. I also created views to match the wireframe given for the project

### CODE REVIEW

File: app/views/posts/index.html.erb [hunk 1]
```diff
@@ -1 +1,19 @@
...
+          <% @post_comments.each do |comment| %>
+            <% if comment.post_id === post.id %>
+            <li class="comment"><%= comment.author.name %>: <%= comment.text %> </li>
+            <% end %>
+          <% end %>
+        </ul>
+      </p>
+    <% end %>
+  </section>
+</main>
```
Comment (KanzaTahreem) [inline app/views/posts/index.html.erb]: - [ ] Good job so far 👍👍 However, please make sure to render 5 most recent comments on post index page as mentioned in the [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_user_all_posts.png) ⛏✨

image

File: app/views/posts/show.html.erb [hunk 2]
```diff
@@ -1 +1,18 @@
...
+  
+  <div class="com_on_Post">
+  <h3>comments:</h3>
+    <ul>
+      <% @post_comments.each do |comment| %>
+        <li><%= comment.author.name %>: <%= comment.text %></li>
+      <% end %>
+    </ul>
+  </div>
+</section>
```
Comment (KanzaTahreem) [inline app/views/posts/show.html.erb]: - [ ] Kindly make sure that the post show page matches the given [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_single_post.png) and move comments and likes to the right as given 🤓📌

image

- [ ] As required for the project, kindly ensure that your design follows the given wireframe, below is how a user card looks 👇. To fix that you need to check the `width` and the `height` of the user photo as this issue appears in all views with user's photo ⛏


image

**Review by KanzaTahreem — CHANGES_REQUESTED** [review]
# Changes Required 🔁

Hi @jonas-45 

Good job so far 👏! I really appreciate the effort you put in!  **_However, there are few this that still need to be fixed. But you are almost there!_** 💪 🏁

so close


## Highlights 🎯

There are some excellent points that you have made perfectly. Keep up the great work! 👏
- ✅ Views are updated well
- ✅ Descriptive PR and README file
- ✅ Correct Linters
- ✅ Well done for using partial

## Required Changes 🛑

_Please check the comments under the review._

## Optional suggestions ❕

N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by t-yanick — APPROVED** [review]
# STATUS: MILESTONE APPROVED :1st_place_medal: 

Hi @jonas-45 , 

### Great job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Requested changes all implemented corectly
:heavy_check_mark: All view pages are as per given wireframe
:heavy_check_mark:  Good commit messages

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by jonas-45** [comment]
Thanks @t-yanick for your time and effort in reviewing my work.

**Pull request merged**